install/requirements_py3.11.txt

```diff
@@ -22,3 +22,4 @@ environs
 pydantic<v2
 scikit-learn
 pymilvus
+clickhouse_connect
```

Comment:
need to add it to `pyproject.toml`
```
all = [
    ...,
    "clickhouse_connect"
]
clickhouse = [ "clickhouse_connect" ]
```
so that users could use "pip install vectordb-bench[all]" or "pip install vectordb-bench[clickhouse]" to install dependencies from `PYPI`.